CarePartner/Models/FolloweeManager.swift

```diff
@@ -230,3 +230,9 @@ extension FolloweeManager: FolloweeDelegate {
         storeFollowee(followee)
     }
 }
+
+extension TPendingInvite {
```

Comment:
This seems fine, however, there is a case where the person doing the inviting is a custodian, are you handling that case?

This is a brief idea of how the backend does it for sending email:
```go
				fullName := invite.Creator.Profile.FullName
				if invite.Creator.Profile.Patient.IsOtherPerson {
					fullName = invite.Creator.Profile.Patient.FullName
				}
```

Reply:
Interesting, so based on this logic it seems the proper order of fallback should be:
1. `invite.creator.profile.patient.fullName`
2. `invite.creator.profile.fullName`
3. `Tidepool User xxxx`

Reply:
@ewollesen curiously, where does the `IsOtherPerson` compute from is that based only on `invite.Creator.Patient` being present?

Reply:
> @ewollesen curiously, what does `IsOtherPerson` compute from? Is that based only on `invite.Creator.Patient` being non-null?
I honestly don't know where that value is set. Maybe @darinkrauss knows? I did some basic grep searches, but didn't find anything. For that matter, if I'm 100% honest, my understanding of that Patient object is largely educated guesswork.

Reply:
It is from well before my time. When you create a new, non-clinician account and you setup data storage, one question it asks is whether the data is for yourself or someone else. If you choose someone else, then IsOtherPerson is true. It is typically for a parent creating an account to manage the data for their child with T1D. Does that make sense?

Reply:
@darinkrauss gotcha! In this case, I think [my assumption](https://github.com/tidepool-org/CarePartner/pull/15#discussion_r1385730879) here should stand.